projects/plugins/jetpack/modules/google-fonts/current/load-google-fonts.php

```diff
@@ -112,7 +112,7 @@ function jetpack_get_theme_fonts_map() {
  */
 function jetpack_register_google_fonts_to_theme_json( $theme_json ) {
 	$google_fonts_data = jetpack_get_google_fonts_data();
-	if ( ! $google_fonts_data ) {
+	if ( ! $google_fonts_data || ! isset( $google_fonts_data['fontFamilies'] ) || ! is_array( $google_fonts_data['fontFamilies'] ) ) {
```

Comment:
Idea: the problematic data is returned by `jetpack_get_google_fonts_data()`, which is used in two call sites:

- `jetpack_get_available_google_fonts_map()` (line 58)
- `jetpack_register_google_fonts_to_theme_json()` (here)

I think it will be even more safe if the `jetpack_get_google_fonts_data()` function itself that verifies if `['fontFamilies']` exists and is an array,  so that we don't have to do this multiple times. Because it's this function that parses the JSON from a remote URL, so it should be also its responsibility to provide the correct shape in the return value.

E.g. one possible "bug" is that in line 58, we didn't add the "is_array" check, but here we added that, which is not symmetric.

So something like:

```
if ( ! isset( $data['fontFamilies'] ) || ! is_array( $data['fontFamilies'] ) {
  return null;
}
return $data;
```

What do you think?